assets/js/theme/common/local-storage-check.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Check if localStorage is available for use
+ * https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API#Testing_for_availability
+ * @param type
+ * @returns boolean
+ */
+export function storageAvailable(type) {
```

Comment:
I think this belongs in stencil utils so it makes it easier for us to make changes in the future.

Reply:
I think since we are just starting to dabble in local storage and we don't yet know exactly what kind of interface we'd want into it, it's totally fine to start out just in cornerstone and then if we have a need for it elsewhere, extract it

Reply:
On the other hand, @junedkazi you make a good point that putting things in stencil-utils makes it easier for those that have forked Cornerstone in private themes to keep things up to date. @bookernath how hard would it be to add this to stencil-utils instead?